system/lib/wasmfs/backends/opfs_backend.cpp

```diff
@@ -28,10 +28,8 @@ void _wasmfs_opfs_get_child(em_proxying_ctx* ctx,
                             int* child_id);
 
 // Create a file under `parent` with `name` and store its ID in `child_id`.
-void _wasmfs_opfs_insert_file(em_proxying_ctx* ctx,
-                              int parent,
-                              const char* name,
-                              int* child_id);
+void _wasmfs_opfs_insert_file(
+  em_proxying_ctx* ctx, int parent, const char* name, int* child_id, int* err);
```

Comment:
clang-format did this. Should I not have run it?

Reply:
Hmm, I run clang-format on the .cpp files in my similar PRs, but this looks unusual. I would have expected the parameters to continue to be vertically aligned.